API/NoDocs/Microsoft.Extensions.Http.Telemetry/TelemetryConstants.cs

```diff
@@ -0,0 +1,16 @@
+// Assembly 'Microsoft.Extensions.Telemetry.Abstractions'
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace Microsoft.Extensions.Http.Telemetry;
+
+public static class TelemetryConstants
+{
+    public const string RequestMetadataKey = "R9-RequestMetadata";
```

Comment:
Remove internal name from key?

Reply:
btw this API review will only show public const values. Is there internal code adding R9 to keys that still exist? I think it would be worth doing an audit. Keys are pseudo-public API (people depend on them not changing).

Reply:
Big review/overhaul in progress.